edk2toolext/edk2_invocable.py

```diff
@@ -102,7 +102,19 @@ def GetWorkspaceRoot(self) -> os.PathLike:
     def GetPackagesPath(self) -> Iterable[os.PathLike]:
         ''' Use the SettingsManager to an iterable of paths to be used as Edk2 Packages Path'''
         try:
-            return list(set(self.PlatformSettings.GetPackagesPath()) - set(self.GetSkippedDirectories()))
+            packages = [x for x in self.PlatformSettings.GetPackagesPath() if x not in self.GetSkippedDirectories()]
+
+            # Temporarily sort, remove duplicates, and match os specific
```

Comment:
can we move this into its own function?  Should this "check for nesting" be in edk2-pytool-library?   Maybe https://github.com/tianocore/edk2-pytool-library/blob/3de82123f055e65c3babbc4e5833121f4215af51/edk2toollib/uefi/edk2/path_utilities.py#L56